hc/api/tests/test_admin.py

```diff
@@ -7,16 +7,21 @@ class ApiAdminTestCase(BaseTestCase):
     def setUp(self):
         super(ApiAdminTestCase, self).setUp()
         self.check = Check.objects.create(user=self.alice, tags="foo bar")
+        self.alice.is_staff = True
+        self.alice.is_superuser = True
+        self.alice.save()
 
-        ### Set Alice to be staff and superuser and save her :)
+        # Set Alice to be staff and superuser and save her :)
 
     def test_it_shows_channel_list_with_pushbullet(self):
         self.client.login(username="alice@example.org", password="password")
 
-        Channel.objects.create(user=self.alice, kind="pushbullet",
-                               value="test-token")
+        a = Channel.objects.create(user=self.alice, kind="pushbullet",
+                                   value="test-token")
+        self.assertTrue("pushbullet", Channel.objects.create(user=self.alice, kind="pushbullet",
```

Comment:
PEP 8 recommends ensuring statements don't go over 79 characters.Refactor this to meet that criterion.

**Resource**: https://goo.gl/iOWVQg